schema/schema.go

```diff
@@ -333,11 +333,16 @@ func (e *Error) Error() string {
 		return ""
 	}
 
-	var multi error
+	errs := []error{}
 	for _, err := range e.Result.Errors() {
-		multi = multierror.Append(multi, fmt.Errorf("%v", err))
+		errs = append(errs, fmt.Errorf("%v", err))
```

Comment:
```suggestion
		errs = errors.Join(errs, err)
```
(this assumes that `.Errors()` returns `error` and not `string` -- otherwise the `fmt.Errorf` is still required.

Reply:
No, unfortunately Result.Errors() returns []ResultError which is neither error nor []error. It implements String() but not Error(), so we need to turn it into a error using fmt.Errorf.